cmd/build_test_matrix/main.go

```diff
@@ -31,24 +32,101 @@ type GithubActionTestMatrix struct {
 	Include []TestSuitePair `json:"include"`
 }
 
+// GithubActionTestSuiteMatrix represents
+type GithubActionTestSuiteMatrix struct {
+	Include []TestSuite `json:"include"`
+}
+
+type TestSuite struct {
+	EntryPoint string `json:"entrypoint"`
+}
 type TestSuitePair struct {
 	Test       string `json:"test"`
 	EntryPoint string `json:"entrypoint"`
 }
 
 func main() {
-	githubActionMatrix, err := getGithubActionMatrixForTests(e2eTestDirectory, getTestToRun(), getTestEntrypointToRun(), getExcludedTestFunctions())
-	if err != nil {
-		fmt.Printf("error generating github action json: %s", err)
-		os.Exit(1)
+	testBySuite, _ := os.LookupEnv("TEST_BY_SUITE")
```

Comment:
maybe we can add a function that returns this as a bool? 

return `"strings.ToLowerCase(testBySuite) == "true"` to be easier to work with.

To be consistent with the other functions maybe we can create a constant for `"TEST_BY_SUITE"` also. WDYT?

Reply:
Nice, thanks for your reviews!